tests/test_article.py

```diff
@@ -608,3 +623,93 @@ def test_crawl():
     assert requests[1].url == "https://example.com/link_1"
     assert requests[2].url == "https://example.com/link_2"
     assert requests[3].url == "https://example.com/link_3"
+
+
+@deferred_f_from_coro_f
+async def test_extract_jobs(zyte_api_server, articles_website):
```

Comment:
Tests in this module have "jobs" in their name; should they be renamed to mention "articles"?